xchange-okex/src/main/java/org/knowm/xchange/okex/v5/OkexAdapters.java

```diff
@@ -240,6 +235,11 @@ public static ExchangeMetaData adaptToExchangeMetaData(
             ? new HashMap<>()
             : exchangeMetaData.getCurrencies();
 
+    String makerFee = "0.5";
```

Comment:
It was already hard-coded in the implementation(line 255 in the same class). I just take it to the initial definition to use in case of any failure or missing data. So compared to the previous version, it seems okay.

Reply:
@earce Will you merge this one? Did my comment suffice?